experimental/traffic-portal/src/app/api/delivery-service.service.ts

```diff
@@ -158,6 +159,17 @@ export class DeliveryServiceService extends APIService {
 		this.deliveryServiceTypes = new Array<Type>();
 	}
 
+	public async getSteering(): Promise<Array<SteeringConfiguration>>;
```

Comment:
This overload is redundant, because the call signature is identical to that of the implementation